cmstestsuite/Tests.py

```diff
@@ -172,7 +186,8 @@
 
     Test('compile-fail',
          task=batch_fileio, filenames=['compile-fail.%l'],
-         languages=COMPILED_LANGUAGES,
+         # PHP's compilation step cannot fail.
```

Comment:
I would elaborate here and say: PHP's compilation step cannot fail as it's just a `cp` command.